projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+# Datagrid - Style API
+
+## CSS Variables
+
+| CSS Variable Name                                     | Description                                                                  |
+| ----------------------------------------------------- | -----------------------------------------------------------------------------|
+| --clr-datagrid-font-color                             | Changes the text color of datagrid.                                          |
+| --clr-datagrid-default-border-color                   | Changes the border color of datagrid when default.                           |
+| --clr-datagrid-icon-color                             | Changes the header icons color of datagrid.                                  |
```

Comment:
Changes the datagrid icons color

// there is nothing about header here, it's used for the action button, detail button and expandable row caret button

Reply:
Yes, changed it as you suggested.